cloud-shell/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM gcr.io/cloudshell-images/cloudshell:latest
```

Comment:
You need to add copyright header to Dockerfile. Look at other Docker files in the repo for example.

Reply:
Got it--added Apache license header!